frontend/public/components/modals/configure-machine-autoscaler-modal.tsx

```diff
@@ -1,141 +1,136 @@
-    this.setState({ maxReplicas });
-  };
+    const changeMaxReplicasBy = (operation) => {
+      setMaxReplicas(maxReplicas + operation);
+    };
 
-  createAutoscaler = (): Promise<K8sResourceKind> => {
-    const {
-      apiVersion,
-      kind,
-      metadata: { name, namespace },
-    } = this.props.machineSet;
-    const { minReplicas, maxReplicas } = this.state;
+    const createAutoscaler = (): Promise<K8sResourceKind> => {
+      const {
+        apiVersion,
+        kind,
+        metadata: { name, namespace },
+      } = props.machineSet;
 
-    const machineAutoscaler = {
-      apiVersion: 'autoscaling.openshift.io/v1beta1',
-      kind: 'MachineAutoscaler',
-      metadata: {
-        name,
-        namespace,
-      },
-      spec: {
-        minReplicas,
-        maxReplicas,
-        scaleTargetRef: {
-          apiVersion,
-          kind,
+      const machineAutoscaler = {
+        apiVersion: 'autoscaling.openshift.io/v1beta1',
+        kind: 'MachineAutoscaler',
+        metadata: {
           name,
+          namespace,
         },
-      },
+        spec: {
+          minReplicas,
+          maxReplicas,
+          scaleTargetRef: {
+            apiVersion,
+            kind,
+            name,
+          },
+        },
+      };
+      return k8sCreate(MachineAutoscalerModel, machineAutoscaler);
     };
 
-    return k8sCreate(MachineAutoscalerModel, machineAutoscaler);
-  };
-
-  submit = (event): void => {
-    event.preventDefault();
-    const { close } = this.props;
-    const promise = this.createAutoscaler();
-    this.handlePromise(promise).then((obj: K8sResourceKind) => {
-      close();
-      history.push(
-        resourcePathFromModel(MachineAutoscalerModel, obj.metadata.name, obj.metadata.namespace),
-      );
-    });
-  };
+    const submit = (event): void => {
+      event.preventDefault();
+      const { close } = props;
+      const promise = createAutoscaler();
+      props.handlePromise(promise, (obj: K8sResourceKind) => {
+        close();
+        history.push(
+          resourcePathFromModel(MachineAutoscalerModel, obj.metadata.name, obj.metadata.namespace),
+        );
+      });
+    };
 
-  render() {
-    const { name } = this.props.machineSet.metadata;
+    const { machineSet, inProgress, errorMessage, cancel } = props;
+    const { t } = useTranslation();
 
     return (
-      <form onSubmit={this.submit} name="form" className="modal-content">
-        <ModalTitle className="modal-header">Create Machine Autoscaler</ModalTitle>
+      <form onSubmit={submit} name="form" className="modal-content">
+        <ModalTitle className="modal-header">
+          {' '}
+          {t('machine-sets~Create MachineAutoscaler')}
+        </ModalTitle>
         <ModalBody>
           <p>
-            This will automatically scale machine set <b>{name}</b>.
+            {t('machine-sets~This will automatically scale machine set')}{' '}
+            <b>{machineSet.metadata.name}</b>.
```

Comment:
This should be a parameter to the message. We probably need to use the `Trans` component since it has markup.